container-search/src/main/java/com/yahoo/prelude/Pong.java

```diff
@@ -47,15 +46,6 @@ public ErrorMessage getError(int i) {
     public int getErrorSize() {
         return errors.size();
     }
-
-    /** 
-     * Returns the package causing this to exist, or empty if none 
-     * 
-     * @deprecated do not use
-     */
-    // TODO: Remove on Vespa 7
-    @Deprecated // OK
-    public Optional<PongPacket> getPongPacket() { return pongPacket; }
```

Comment:
This is removed on Vespa 7 but can't be removed on master.

Reply:
Why not ? prelude.is not PublicApi

Reply:
Hm, ok then.